website/docs/guides/getting-started.html.markdown

```diff
@@ -0,0 +1,592 @@
+
+While you could use `kubectl` or similar CLI-based tools mapped to API calls
+to manage all K8S resources described in YAML files,
+orchestration with Terraform presents a few benefits.
+
+ - [HCL](https://www.terraform.io/docs/configuration/syntax.html) - same
+    language for both lower layer with underlying infrastructure (compute)
+    and scheduling layer
+ - drift detection - `terraform plan` will always present you the difference
+    between reality at a given time and config you intend to apply.
+ - full lifecycle management - Terraform doesn't just initially create resources,
+    but offers a single command for creation, update, and deletion of tracked
+    resources without needing to inspect the API to identify those resources.
+ - synchronous feedback - While asynchronous behaviour is often useful,
+    sometimes it's counter-productive as the job of identifying operation result
+    (failures or details of created resource) is left to the user. e.g. you don't
+    have IP/hostname of load balancer until it has finished provisioning,
+    hence you can't create any DNS record pointing to it.
+ - [graph of relationships](https://www.terraform.io/docs/internals/graph.html) -
+    Terraform understands relationships between resources which may help
+    in scheduling - e.g. if a Persistent Volume Claim claims space from
+    a particular Persistent Volume Terraform won't even attempt to create
+    the PVC if creation of the PV has failed.
+
+## Provider Setup
+
+The easiest way to configure the provider is by creating/generating a config
+in a default location (`~/.kube/config`). That allows you to leave the
+provider block completely empty.
+
+```hcl
+provider "kubernetes" {}
+```
+
+If you wish to configure the provider statically you can do so
+
+```hcl
+provider "kubernetes" {
+  host     = "https://104.196.242.174"
+  username = "ClusterMaster"
+  password = "MindTheGap"
+
+  client_certificate     = "${file("~/.kube/client-cert.pem")}"
+  client_key             = "${file("~/.kube/client-key.pem")}"
+  cluster_ca_certificate = "${file("~/.kube/cluster-ca-cert.pem")}"
+}
+```
+
+After specifying the provider we may now run the following command
+to pull down the latest version of the Kubernetes provider.
+
+```
+$ terraform init
+
+
+Initializing provider plugins...
+- Downloading plugin for provider "kubernetes"...
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
+```
+
+## Scheduling a Simple Application
+
+The bread and butter of any K8S app is [a Pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/#what-is-a-pod).
```

Comment:
Can we find a different way to say "bread and butter", since it's an idiom that might not be familiar to all readers?

I'm not sure exactly what was intended here but I'd consider something like:

> The main object in any Kubernetes app is its [Pod](https://kubernetes.io/docs/concepts/workloads/pods/pod/#what-is-a-pod), which consists of one or more containers that are placed on cluster nodes based on CPU and memory availability.

---

Also (this feedback applies generally for the whole docs, really) I think we should stick to writing out "Kubernetes" in full for consistency, since I think K8S is more of a "nickname".